py-polars/polars/internals/expr.py

```diff
@@ -893,6 +893,9 @@ def min(self) -> "Expr":
     def sum(self) -> "Expr":
         """
         Get sum value.
+
+        Note that dtypes in {Int8, UInt8, Int16, UInt16} are cast to
```

Comment:
This seems like a good candidate for a `Notes` section: https://numpydoc.readthedocs.io/en/latest/format.html#notes

Reply:
Good one, coming up in next PR.